src/plugins/ui-router-translations/ui-router-translations-umd.js

```diff
@@ -0,0 +1,54 @@
+const { createFilter } = require('rollup-pluginutils');
+const _ = require('lodash');
+const fs = require('fs');
+const glob = require('glob');
+const mkdirp = require('mkdirp');
+const path = require('path');
+
+const translationOutput = {};
+const translationTools = require('@ovh-ux/translations-build-tools');
+const utils = require('./utils');
+const config = require('../../config');
+
+const stateHander = (source, options) => {
+  const { translations, errors } = utils.listTranslations(source, process.cwd(), options);
+  const relativePath = path.relative(process.cwd(), options.translationsRoot);
+
+  translations.forEach((t) => {
+    // export xml translations to js
+    glob.sync(path.resolve(relativePath, t, '*.xml')).forEach((translationPath) => {
+      const xmldata = fs.readFileSync(translationPath);
+      const name = path.basename(translationPath).replace(/\.xml$/, '.json');
+      translationOutput[name] = translationOutput[name] || {};
+      translationOutput[name] = _.merge(
+        translationOutput[name],
+        JSON.parse(translationTools.translations(xmldata)),
+      );
+    });
+  });
+
+  return { result: source, errors };
```

Comment:
We should call this variable "result" or "source", but not at the same time :D

I think we loose some logic to convert source into result, result into data, data into source, source into result etc etc etc, be smart ! :)